source/agora/consensus/data/Transaction.d

```diff
@@ -377,6 +377,21 @@ unittest
     assert(expected2 == tx_freeze_hash, expected2.toString());
 }
 
+// The status of the result of posting a transaction
+public enum TransactionStatus : uint
```

Comment:
I couldn't directly relate this naming and enum with `postTransaction` and `TransactionResult`, how about renaming it as `Status` inside `TransactionResult`?

```
public struct TransactionResult
{
   public enum Status
   {...
```

Reply:
Good suggestion! Will apply it.